torchtune/training/checkpointing/_checkpointer.py

```diff
@@ -341,7 +344,7 @@ def save_checkpoint(
             _ = state_dict.pop(training.ADAPTER_KEY, None)
             _ = state_dict.pop(training.ADAPTER_CONFIG, None)
             output_path = Path.joinpath(
-                self._output_dir, RECIPE_STATE_DIRNAME, "recipe_state.pt"
+                self._output_dir, f"epoch_{epoch}", "recipe_state.pt"
```

Comment:
Why just epoch here?

Reply:
It's a debugger torchtune checkpointer, not a HF one